docs/resources/Guild.md

```diff
@@ -348,7 +349,7 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
 | nick?                         | ?string                                         | this user's guild nickname                                                                                                                                                                                                           |
 | avatar?                       | ?string                                         | the member's [guild avatar hash](#DOCS_REFERENCE/image-formatting)                                                                                                                                                                   |
 | roles                         | array of snowflakes                             | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids                                                                                                                                                                     |
-| joined_at                     | ISO8601 timestamp                               | when the user joined the guild                                                                                                                                                                                                       |
+| joined_at                     | ?ISO8601 timestamp                              | when the user joined the guild                                                                                                                                                                                                       |
```

Comment:
this is a breaking change that will cause issues for bots, shouldn't this be part of a new api version?